plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -126,6 +125,21 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
     if (min === undefined) min = 0;
     if (max === undefined) max = 1;
   }
+
+  const xAxisFormatter = (value: number) => {
```

Comment:
As we are changing the value around, I think defining the return type/s of the function would be great

Reply:
But hasn't typescript already infer that already?